docs/scripts.js

```diff
@@ -284,6 +284,39 @@ $(function () {
         saveState();
     }
 
+    function parseArduinoCode(text) {
+        const UINT64_REGEX = /^\s*const\s+uint64_t\s+(\w+)\s*\[\s*\]\s*=\s*{((?:.|\n)*)}\s*;\s*(?:const\s+(\w+)\s+\1_LEN\s*=\s*sizeof\s*\(\s*\1\s*\)\s*\/\s*8\s*;\s*)?$/;
+        const UINT8_REGEX = /^\s*const\s+(?:uint8_t|byte)\s+(\w+)\s*\[\s*\]\s*\[\s*8\s*\]\s*=\s*{((?:.|\n)*)}\s*;\s*(?:const\s+(\w+)\s+\1_LEN\s*=\s*sizeof\s*\(\s*\1\s*\)\s*\/\s*8\s*;\s*)?$/;
+
+        let matrices;
+
+        let match;
+        if (match = UINT64_REGEX.exec(text)) {
+            matrices = Array.from(match[2].matchAll(/0x[0-9a-zA-Z]{16}/g)).map(match => match[0].substring(2));
+        } else if (match = UINT8_REGEX.exec(text)) {
+            let bytesStr = Array.from(match[2].matchAll(/0b[01]{8}/g)).map(match => match[0]);
```

Comment:
At first sight this line can be changed to support both prefixes and to extract only bits content.
`let bytesStr = Array.from(match[2].matchAll(/(?:0b|0B|B)([01]{8})/g)).map(match => match[1]);`
So it will not be needed to do `b.substring(2)` in the line below.

Reply:
Indeed, I'll change that!